pkg/engine/const.go

```diff
@@ -199,8 +199,8 @@ const (
 const (
 	heapsterAddonSourceFilename                    string = "kubernetesmasteraddons-heapster-deployment.yaml"
 	heapsterAddonDestinationFilename               string = "kube-heapster-deployment.yaml"
-	metricsServerAddonSourceFilename               string = "kubernetesmasteraddons-metrics-server-deployment.yaml"
-	metricsServerAddonDestinationFilename          string = "kube-metrics-server-deployment.yaml"
+	metricsServerAddonSourceFilename               string = "metrics-server.yaml"
```

Comment:
At some point maybe we should fold source and destination into just a single `metricsServerAddonFilename`, for example.

Reply:
yeah, that would probably be easier, just not changing that yet becasue reasons